packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -330,7 +330,9 @@ module.exports = (
     }
 
     async function getHTML(markdownNode) {
-      const cachedHTML = await cache.get(htmlCacheKey(markdownNode))
+      const shouldCache = !!markdownNode && markdownNode.internal
```

Comment:
```suggestion
      const shouldCache = markdownNode && markdownNode.internal
```

Do we need the double _not_ operator here?